yarn-project/prover-node/src/prover-node.ts

```diff
@@ -115,32 +118,40 @@ export class ProverNode implements ClaimsMonitorHandler, EpochMonitorHandler {
    */
   async handleEpochCompleted(epochNumber: bigint): Promise<void> {
     try {
+      // Construct a quote for the epoch
       const blocks = await this.l2BlockSource.getBlocksForEpoch(epochNumber);
       const partialQuote = await this.quoteProvider.getQuote(Number(epochNumber), blocks);
       if (!partialQuote) {
         this.log.verbose(`No quote produced for epoch ${epochNumber}`);
         return;
       }
 
+      // Ensure we have deposited enough funds for sending this quote
+      await this.bondManager.ensureBond(partialQuote.bondAmount);
+
+      // Assemble and sign full quote
       const quote = EpochProofQuotePayload.from({
         ...partialQuote,
         epochToProve: BigInt(epochNumber),
         prover: this.publisher.getSenderAddress(),
         validUntilSlot: partialQuote.validUntilSlot ?? BigInt(Number.MAX_SAFE_INTEGER), // Should we constrain this?
       });
       const signed = await this.quoteSigner.sign(quote);
+
+      // Send it to the coordinator
       await this.sendEpochProofQuote(signed);
```

Comment:
Does the coordinator also send quotes to a json-rpc endpoint? For provernet testing purposes, we don't expect provers to get quotes via the p2p.

Reply:
In the provernet scenario, the coordinator is the single sequencer that will be running. And yes, it'll be sent via json-rpc, and not p2p.